crates/shared/src/sources/balancer_v2/pool_fetching.rs

```diff
@@ -86,33 +80,6 @@ impl BalancerPoolEvaluating for WeightedPool {
     }
 }
 
-#[derive(Clone, Debug, PartialEq)]
```

Comment:
This just moved to `shared::sources::balancer_v2::pools::stable` without any change.